### PR TITLE
SPCTR-3630: Twenty sixteen theme Compatibility

### PR DESCRIPTION
**Pull request by yash-tingre-10**
### Description
Twenty sixteen theme Compatibility

### TestCases
https://bsf.d.pr/v/X4X4Dn

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: compatibility/class-uagb-twenty-sixteen-compatibility.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		}
+		/**
+		 * Generates stylesheet and appends in head tag.
+		 *
+		 * @since 2.11.3
+		 * @return void
+		 */
+		public function generate_stylesheet() {
+
+			if ( is_home() ) {
```
Comment (AbhijitNage123) [inline compatibility/class-uagb-twenty-sixteen-compatibility.php:54]: @yash-tingre-10 Is issue is only on home page? What about other pages?

Comment (yash-tingre-10) [reply compatibility/class-uagb-twenty-sixteen-compatibility.php:54]: It's only for home page or the page which is set as post page same as in twenty seventeen theme

Comment (AbhijitNage123) [reply compatibility/class-uagb-twenty-sixteen-compatibility.php:54]: Okay

File: README.md [hunk 2]
```diff
@@ -214,6 +214,8 @@ When you use the Spectra along with the free Astra theme, you get a huge library
 Our external packages use [Rating Star Component](https://github.com/n49/react-stars) that are distributed under the terms of the ISC. While [Slick](https://github.com/akiran/react-slick), [Lottie](https://github.com/chenqingspring/react-lottie) and [Striptags](https://github.com/ericnorris/striptags) is distributed under the terms of the MIT.
 
 ## Changelog ##
+### x.x.x ###
+* Fix: Icon - Resolved an issue wherein the icon would appear enlarged in the Twenty Sixteen theme.
```
Comment (AbhijitNage123) [inline README.md:218]: * Fix: Twenty Sixteen Theme - Resolved an issue wherein the icon would appear enlarged.

**Review by AbhijitNage123 — APPROVED** [review]
Look Good To Me

File: README.md (hunk 2 above)
Comment (yash-tingre-10) [reply README.md:218]: I'll verify it from zane and add it

File: compatibility/class-uagb-twenty-sixteen-compatibility.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+		 * Member Variable
+		 *
+		 * @var UAGB_Twenty_Sixteen_Compatibility
+		 */
+		private static $instance;
+
+		/**
+		 *  Initiator
+		 *
+		 * @since 2.11.3
```
Comment (adi3890) [inline compatibility/class-uagb-twenty-sixteen-compatibility.php:29]: make this as x.x.x wherever copied.
self review if required

File: README.md (hunk 2 above)
Comment (yash-tingre-10) [reply README.md:218]: Got this updated change log from @bsf-zanev 
* Fix: Global - Resolved an issue wherein all icons would appear enlarged in the Twenty Sixteen Theme.

File: compatibility/class-uagb-twenty-sixteen-compatibility.php (hunk 3 above)
Comment (yash-tingre-10) [reply compatibility/class-uagb-twenty-sixteen-compatibility.php]: Done

File: readme.txt [hunk 4]
```diff
@@ -214,6 +214,8 @@ When you use the Spectra along with the free Astra theme, you get a huge library
 Our external packages use [Rating Star Component](https://github.com/n49/react-stars) that are distributed under the terms of the ISC. While [Slick](https://github.com/akiran/react-slick), [Lottie](https://github.com/chenqingspring/react-lottie) and [Striptags](https://github.com/ericnorris/striptags) is distributed under the terms of the MIT.
 
 == Changelog ==
+= x.x.x =
```
Comment (adi3890) [inline readme.txt:217]: @yash-tingre-10 i see 2 new changelogs can you merge them and make one.

File: readme.txt [hunk 5]
```diff
@@ -216,9 +216,11 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 == Changelog ==
 
 = x.x.x =
+* Fix: Global - Resolved an issue wherein all icons would appear enlarged in the Twenty Sixteen Theme.
 * Fix: Image Gallery - Resolved stacking issue on Safari during the initial load when choosing the masonry layout.
 * Fix: Popup Builder - Resolved an issue wherein dynamic blocks inside the popups would not be visible on the fronted.
 
+
```
Comment (adi3890) [inline readme.txt:223]: Please remove this extra line in changelog.

Comment (yash-tingre-10) [reply readme.txt]: Removed the line

**Pull request merged**